test

AluTempla opened the pull request:
**Reasons for making this change:**

_TODO_

**Links to documentation supporting these rule changes:** 

_TODO_

If this is a new template: 

 - **Link to application or project’s homepage**: _TODO_

jjst committed:
Symlink Kotlin.gitignore to Java.gitignore

jpdillingham committed:
Add exclusion for OpenCover UI results

mansourcodes committed:
ignore logs on /administrator/logs/*

weinihou committed:
add MANIFEST to Python.gitignore

add MANIFEST which is GENERATED by distutils

Reasons for making this change:

if you use distutils.core to setup, the file MANIFEST is generated by distutils when run 'python setup.py sdist'

Coeur committed:
clarifying when ignoring is not required

jwg4 committed:
Add venv.bak - created by terrarium

Terrarium backs up old virtual environments by moving them to this dir.

jwg4 committed:
Add env.bak, re-arrange

soreng committed:
Added link to source

Commit:
Add extended DVI (XDV) to TeX.gitignore

Commit:
.DS_Store is a file name, not an extension

sobolevn committed:
Updates Elixir gitignore file

Official `elixir`'s `gitignore` after running `mix new` contains these two lines:

> # Where 3rd-party dependencies like ExDoc output generated docs.
> /doc
> # Ignore .fetch files in case you like to edit your project deps locally.
> /.fetch

They are missing at the moment.

yetanotherchris committed:
Update Terraform.gitignore

Added .tfvars file so for example AWS credentials aren't committed.

Xeio committed:
Fix comments on same line causing ignore to break

shiftkey committed:
Merge pull request #2451 from Xeio/patch-1

Fix comments on same line causing ignore to break

CodeNotFoundException committed:
Adds .gitignore rules for AxoCover

AxoCover is a Code Coverage Tool. It creates a **.axoCover** folder where are created:
- settings in settings.json
- sub-folders into **run** folder which contain code coverage results.

All content into **.axoCover** folder must be ignored except the **settings.json** file.

shiftkey committed:
Merge pull request #2454 from CodeNotFoundException/patch-1

Adds .gitignore rules for AxoCover

pei-lun committed:
'*.tfstate.*.backup' added to Terraform.gitignore

shiftkey committed:
Merge pull request #2455 from pei-lun/terraform.gitignore

'*.tfstate.*.backup' added to Terraform.gitignore

shiftkey committed:
Merge pull request #1880 from jjst/kotlin

Add gitignore for Kotlin

timsueberkrueb committed:
Add QML and JS cache files to Qt.gitignore

shiftkey committed:
Merge pull request #2456 from tim-sueberkrueb/patch-2

Add QML and JS cache files to Qt.gitignore

wirgen committed:
Add default file, README in libraries and translations in Drupal

shiftkey committed:
Merge pull request #2458 from wirgen/master

Add default file, README in libraries and translations in Drupal

FabrizioBrancati committed:
Updating Swift.gitignore with Package.resolved

shiftkey committed:
Merge pull request #2461 from FabrizioBrancati/master

Updating Swift.gitignore with Package.resolved

renjithsraj committed:
updated static and media files in django section

denisdefreyne committed:
Nanoc: Ignore tmp/nanoc/, not tmp/

shiftkey committed:
Merge pull request #2465 from ddfreyne/tmp-nanoc

Nanoc: Ignore tmp/nanoc/, not tmp/

rdela committed:
Node - update Compiled binary addons link to https

Node.gitignore build/Release Compiled binary addons: http to https
https://nodejs.org/api/addons.html

shiftkey committed:
Merge pull request #2467 from rdela/patch-1

Node - update Compiled binary addons link to https

fabioricali committed:
Adds some files

shiftkey committed:
Merge pull request #2470 from fabioricali/patch-1

Adds some files

shiftkey committed:
Merge pull request #2422 from danilcha/master

.DS_Store is a file name, not an extension

shiftkey committed:
Merge pull request #2429 from yetanotherchris/patch-1

Update Terraform.gitignore - ignore terraform.tfvars

shiftkey committed:
Merge pull request #2419 from soreng/patch-1

Added link to source

shiftkey committed:
Merge pull request #2425 from sobolevn/patch-1

Updates Elixir gitignore file

shiftkey committed:
Merge pull request #2420 from niklasholm/master

Add extended DVI (XDV) to TeX.gitignore

shiftkey committed:
Merge pull request #2397 from weinihou/master

add MANIFEST to Python.gitignore

shiftkey committed:
Merge pull request #2416 from jwg4/patch-2

Add names of backup folders created by terrarium

shiftkey committed:
Merge pull request #2410 from Coeur/patch-2

clarifying when ignoring is not required

shiftkey committed:
Merge pull request #2332 from alnassre/patch-1

ignore logs on /administrator/logs/*

shiftkey committed:
Merge pull request #2328 from jpdillingham/opencover-visualstudio

Add exclusion for OpenCover UI results

Ebola16 committed:
Remove double-spaced lines

Ayase-252 committed:
Fix failure to ignore simulink autosave file

`.autosave` does not match the name of autosave file generated by
simulink. A leading asterisk(*) is needed to match such name.

shiftkey committed:
Merge pull request #2474 from Ebola16/master

Remove double-spaced lines

shiftkey committed:
Merge pull request #2475 from Ayase-252/master

Fix failure to ignore Simulink autosave files

Commit:
Update OCaml.gitignore

Merlin is a code completion for OCaml in Vim and Emacs. Such configuraing file is common

shiftkey committed:
Merge pull request #2477 from Ckins/patch-1

Update OCaml.gitignore

Commit:
Update Autotools.gitignore

add files generated by the autotool m4.

shiftkey committed:
Merge pull request #2480 from Ckins/master

Update Autotools.gitignore

jimbob7198 committed:
Add exclusion of Visual Studio trace files (*.e2e)

shiftkey committed:
Merge pull request #2464 from Cadmus/master

updated static and media files in django section

shiftkey committed:
Merge pull request #2482 from jimbob7198/patch-1

Add exclusion of Visual Studio trace files (*.e2e)

friederbluemle commented:
@AluTempla I think you might have opened this PR on my fork by accident. Please check the official repo if you want to contribute: https://github.com/github/gitignore

AluTempla commented:
Sorry, im very new to all of this.

On Sep 20, 2017 11:41 PM, "Frieder Bluemle" <notifications@github.com>
wrote:

> @AluTempla <https://github.com/alutempla> I think you might have opened
> this PR on my fork by accident. Please check the official repo if you want
> to contribute: https://github.com/github/gitignore
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/friederbluemle/gitignore/pull/1#issuecomment-331042590>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AeeuqSahf_kXClY3QHA2pnk6MYctLwLZks5skdr-gaJpZM4PYqjg>
> .
>

friederbluemle commented:
No problem, we all were at some point :)
You can read some basics about pull requests here: https://help.github.com/articles/about-pull-requests/
Let me know if I can help with anything.

malhobayyeb committed:
Commenting out Android keystore files

Android keystore files are critical and losing them is destructive.

They should be commented out by default, the user should explicitly uncomment their line if they intend not to include them into their repository.

shiftkey committed:
Merge pull request #2496 from malhobayyeb/comment-out-android-keys

Commenting out Android keystore files

flowonyx committed:
Fixed typos in README.md

There were just a couple of typos in the README that I fixed.

shiftkey committed:
Merge pull request #2501 from flowonyx/patch-1

Fixed typos in README.md

Commit:
Ignore Nuget package directory name casing

It looks like VS2017 changed it's nuget package directory name to be capitalized.
When using Ubuntu for Windows, directory name casing of NuGet package folder becomes important and detected as a change by git,. This change is to ignore regardless of casing.

shiftkey committed:
Merge pull request #2503 from synepis/patch-1

Ignore Nuget package directory name casing

Yousha committed:
Change desktop.ini to [Dd]esktop.ini

shiftkey committed:
Merge pull request #2504 from Yousha/master

Windows.gitignore - Change desktop.ini to [Dd]esktop.ini

bndw committed:
Remove .glide/ ignore

This was a workaround for https://github.com/Masterminds/glide/issues/736

Signed-off-by: bndw <benjamindwoodward@gmail.com>

cliffordp committed:
Fix link to go to current anchor

hantuzun committed:
Update Terraform.gitignore

Improvements: 

 * `.terraform` directories could be created in any subdirectory when user runs `terraform init`.
 * Quite an interesting variations of`.tfstate` files are created by Terraform such as `.*.tfstate.d` or  `.*.tfstate.lock.info`. It's best not to name any file with `*.tfstate.*` pattern.
 * `*.tfvars` files often contain private data and not supposed to be committed.

shiftkey committed:
Merge pull request #2506 from cliffordp/patch-1

Fix link to go to current anchor

shiftkey committed:
Merge pull request #2509 from hantuzun/patch-1

Update Terraform.gitignore

jsoref committed:
spelling: intellij

jsoref committed:
spelling: utils

shiftkey committed:
Merge pull request #2512 from jsoref/spelling

Spelling

Commit:
Revert "Remove Composer's file and directory."

This reverts commit 5b83681411d95d9302a3e0414efe82c6b4549d26.

ButchDean committed:
Ignore Visual Studio 2017 generated files.

Noctis- committed:
Update comment

xeoneux committed:
Update VisualStudio.gitignore

shiftkey committed:
Merge pull request #2519 from xeoneux/patch-1

Update VisualStudio.gitignore

shiftkey committed:
Merge pull request #2517 from Noctis-/patch-1

Update comment

halbich committed:
Include Plugins folder

shiftkey committed:
Merge pull request #2524 from halbich/unreal-plugins-directories

UnrealEngine - Include Plugins folder

shiftkey committed:
Merge pull request #2516 from ButchDean/master

Ignore Visual Studio 2017 C++ app generated files.

shiftkey committed:
Merge pull request #2505 from bndw/go-remove-glide

Remove .glide/ ignore

shiftkey committed:
Merge pull request #2513 from lushc/feature/ignore-vendors

Revert "Remove Composer's file and directory."

Commit:
Update VisualStudio.gitignore (#2510)

* Update VisualStudio.gitignore

cmower committed:
TeX.gitignore modification for Emacs users (#2491)

* without . the auto directory is still considered for commits

* small addition that ensures all Emacs-Lisp files (.el files) are ignored

augustoproiete committed:
Add sccprj/ folder to global MATLAB.gitignore (#2493)

Commit:
added logs and devel directories to .gitignore (#2497)

thiminhnhut committed:
Add *.sta (standalone packages) for TeX (#2484)

matzar committed:
Update Unity.gitignore (#2502)

Prevents directories: `Library`
`Temp`
`Obj`
`Build`
`Builds`
`Assets/AssetStoreTools`
from being included in the remote if the project sits in the `Assets` folder.

jrreed committed:
Ignore Unity generated '*.pdb.meta' files for '*.pdb' files (#2486)

RyanGlScott committed:
Add cabal.project.local~ to Haskell.gitignore (#2479)

t-my committed:
Add ignore for Azure Stream Analytics local run output folder (#2498)

matzar committed:
Fixing .gitignore (#2531)

* Update Unity.gitignore

Prevents directories: `Library`
`Temp`
`Obj`
`Build`
`Builds`
`Assets/AssetStoreTools`
from being included in the remote if the project sits in the `Assets` folder.

* removed wildcards from Unity's .gitignore

tasken committed:
Update Jekyll.gitignore (#2540)

Add .jekyll-cache/ as per envygeeks/jekyll-assets

enricosada committed:
ignore MSBuild Binary and Structured Log (#2549)

ref http://msbuildlog.com/

richarth committed:
Ignoring the models builder out of date flag (#2543)

The out of date models flag is automatically generated by the system, we don't want this in Git

Commit:
Ignore node_modules (#2538)

Rails 5 uses npm to handle javascript dependancies, node_modules should not be checked in to git.

sllroberts committed:
Update Matlab.gitignore (#2533)

Added extensions for prepackaged app and toolbox file, pregenerated helpsearch folders.

Commit:
Add "cmake-build-release/" to Jetbrains.gitignore (#2552)

mostafasoufi committed:
Added Must Use Plugins to gotignore (#2553)

https://codex.wordpress.org/Must_Use_Plugins

timonsku committed:
Added a few common CAM file extensions (#2555)

wolf99 committed:
Add *.snk strong name key files (#2483)

* Add *.snk strong name key files

Strong name key files shouldn't be included in a repository AFAIK. they are intended to sign build output to verify that it comes from the correct publisher. having an *.snk in a repository would allow anyone to produce build as if they were the original publisher.

I guess some OSS projects might like to have *.snk files in their repos but that would be an exception to the rule.

* Make use of *.snk optional

Add note explaining use.

* Reduce explanation

Just use a link instead

rafiw committed:
Update Eclipse.gitignore (#2556)

* Update Eclipse.gitignore

* Update Eclipse.gitignore

mxschmitt committed:
Go: Add '*.exe~' to .gitignore

When you build a Go executable, which does already exists and is in use on Windows (e.g. running) then the output binary name will be extended by a '~' prefix.
https://github.com/golang/go/blob/558eeb2d850d064b0b02b65a7bf3af6c108c248d/src/cmd/go/internal/work/exec.go#L1165-L1170

shiftkey committed:
Merge pull request #2561 from maxibanki/patch-1

Go: Add '*.exe~' to .gitignore

rgbkrk committed:
Update Node.gitignore

shiftkey committed:
Merge pull request #2562 from rgbkrk/patch-1

Update Node.gitignore

mbukejlovic committed:
Fixed the unignore of folder 'packages'

Considering the VisualStudio gitignore file contains a line "**/[Pp]ackages/*", these 2 lines from Umbraco gitignore never get applied properly to the desired files, even though Umbraco lines were added after the VS lines. I guess the VS's line is more specific, so it gets a priority of some sort.

shiftkey committed:
Merge pull request #2572 from mbukejlovic/patch-1

Fixed the unignore of folder 'packages'

neetrath committed:
Update Android.gitignore

Add fastlane in Android gitignore

shiftkey committed:
Merge pull request #2578 from neetrath/patch-2

Update Android.gitignore

meneguzzi committed:
Update TeX.gitignore

Added a temporary file used in MacTeX

shiftkey committed:
Merge pull request #2579 from meneguzzi/patch-1

Update TeX.gitignore

felixdivo committed:
Added ignore for files created by `catkin_make_isolated`

Simply execute `catkin_make_isolated` and `build` and `devel` will have different names, but should be ignored as well.

Commit:
Added "ServiceFabricBackup/"

The latest update of Visual Studio 17 (15.5.5) introduces a new backup folder called "ServiceFabricBackup" before upgrading service fabric applications.

chalin committed:
Add `.dart_tool/`

Followup to https://github.com/dart-lang/site-www/issues/444#issuecomment-346560788

shiftkey committed:
Merge pull request #2593 from chalin/patch-1

Add `.dart_tool/`

shiftkey committed:
Merge pull request #2589 from komaflash/master

Added "ServiceFabricBackup/"

shiftkey committed:
Merge pull request #2587 from felixdivo/patch-1

Added ignore for files created by `catkin_make_isolated`

raimon49 committed:
Add ignore '.pytest_cache' directory in Unit test section

shiftkey committed:
Merge pull request #2594 from raimon49/add-ignore-cache-dir-in-unit-test-section

Add ignore '.pytest_cache' directory in Unit test section

Commit:
Fixes  WebStorm capitalization in comment

On line 1, WebStorm is capitalized the same way as the other IDE names.

shiftkey committed:
Merge pull request #2596 from nfcopier/patch-1

Fixes  WebStorm capitalization in comment

DKroot committed:
.idea/**/datasources.xml is now safe to store

Since IDEA 14 (2014), IDEA and other contemporaneous JebBrains IDEs don't store sensitive info in `datasources.xml` and it makes sense to version control to share project data sources among the team. `dataSources.local.xml` is user-specific: contain usernames and could be sensitive from a security perspective, albeit passwords could be externalized to e.g. Mac Keychain.

References: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839, https://youtrack.jetbrains.com/issue/IDEA-127105

shiftkey committed:
Merge pull request #2597 from DKroot/patch-1

.idea/**/datasources.xml is now safe to store

davidebeatrici committed:
Qt: Correct typo "QtCtreator" to "QtCreator"

dgabbe committed:
Exclude secret deployment key in rsconnect/

Exclude secret deployment key in rsconnect/, see https://shiny.rstudio.com/articles/shinyapps.html for details.

shiftkey committed:
Merge pull request #2603 from dgabbe/dgabbe-r-rsconnect

Exclude secret deployment key in rsconnect/

davidebeatrici committed:
Qt: Remove extra empty lines

shiftkey committed:
Merge pull request #2601 from davidebeatrici/qt-typo-fix

Qt: typo fix and extra empty lines removal

CKGrafico committed:
Remove typings ignore

Nobody is using v1 of typings anymore on new projects and 'typings' is the recommended name for the folders of your custom types.
On the other hand the official Visual Studio ignore is not ignoring this folder.

shiftkey committed:
Merge pull request #2608 from CKGrafico/patch-1

Remove typings ignore

BlueNalgene committed:
Added htlatex for TeX.gitignore

kevmoo committed:
Fix URL in comments

Points to valid URI

shiftkey committed:
Merge pull request #2617 from kevmoo/patch-1

Dart.gitignore: Fix URL in comments

shiftkey committed:
Merge pull request #2616 from BlueNalgene/LaTeX_htlatex

Added htlatex for TeX.gitignore
